pkg/handlers/login.go

```diff
@@ -110,7 +110,18 @@ func (h *Handler) Login(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	loginResponse.Token = fmt.Sprintf("Bearer %s", signedJWT)
```

Comment:
Let's remove the `Token` field from the login response type?

Reply:
Addressed in PR https://github.com/replicatedhq/kots/pull/3846